open_mastr/soap_api/download.py

```diff
@@ -909,20 +897,20 @@ def additional_data(self, technology, unit_ids, data_fcn, timeout=10):
 
         return data, data_missed
 
-    def _retrieve_data_in_single_process(self, prepared_args, data_fcn, technology):
+    def _retrieve_data_in_single_process(self, prepared_args, data_fcn, data):
```

Comment:
The renaming here leads to problems with the console output, due to the method internal variable `data`.
![grafik](https://user-images.githubusercontent.com/54852694/186889927-0cee4a3f-f62d-4134-a9c5-d86453d8b52b.png)
I'll push a commit to fix it.

Reply:
Nice that you found that